src/__test__/pages/experiments/[experimentId]/data-processing/index.test.jsx

```diff
@@ -127,7 +134,13 @@ describe('DataProcessingPage', () => {
   const experimentData = {};
 
   it('renders correctly', () => {
```

Comment:
[sand] maybe we could add a more descriptive name now that it has a different initial state, something like "renders correctly with successful pipeline status"  or "renders disabled run data processing button on successful pipeline status"

Reply:
Changed to make it more descriptive